Geometry/HcalAlgo/plugins/dd4hep/HcalDD4HepHelper.h

```diff
@@ -0,0 +1,8 @@
+#ifndef HcalAlgo_HcalDD4HepHelper_h
+#define HcalAlgo_HcalDD4HepHelper_h
+#include "DD4hep/DD4hepUnits.h"
+
+namespace HcalDD4HepHelper {
+  const double convert2mm(double length) { return (length / dd4hep::mm); }
+}  // namespace HcalDD4HepHelper
```

Comment:
I suggested this function a couple of months ago, but Vladimir thought it wasn't necessary. I would put it into `DataFormats/Math/interface/DD4hepUnits.h`. It could be made faster:
```
template <class NumType>
    inline constexpr NumType convert2mm(NumType length)
{ return (length / dd4hep::mm); }
```